### PR TITLE
Improve monitoring documentation

### PR DESCRIPTION
**Pull request by lukebakken**
Please see the following story: #150650781

### CODE REVIEW

File: site/how.xml [hunk 1]
```diff
@@ -210,74 +210,28 @@ limitations under the License.
...
-      <a href="http://del.icio.us/alexisrichardson/rabbitmq+management">RabbitMQ Management</a> and
-      <a href="http://del.icio.us/alexisrichardson/rabbitmq+monitoring">RabbitMQ Monitoring</a> on del.icio.us<br />
-      More links to what others are doing to manage and monitor RabbitMQ.
-    </li>
+        <li>
+          <a href="https://github.com/rabbitmq/chef-cookbook">Chef Cookbook</a><br />
+          A cookbook for managing RabbitMQ with Chef.
+        </li>
+        <li>
+          <a href="https://forge.puppet.com/puppetlabs/rabbitmq">Puppet Module</a><br />
```
Comment (acogoluegnes) [inline site/how.xml]: This one has been deprecated last week, new one is: https://forge.puppet.com/puppet/rabbitmq

File: site/monitoring.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <p>
+        RabbitMQ metrics are made available through the <a href="management.html#http-api">HTTP API</a>
+        or via several <a href="man/rabbitmqctl.1.man.html"><code>rabbitmqctl</code></a> commands. It is
+        recommended to collect metrics at 60 second intervals because more frequent collection may place
+        too much load on the RabbitMQ server and negatively affect performance.
+      </p>
+      <p>
+        <table>
+          <thead>
+            <tr><td>Metric</td><td>JSON field name</td></tr>
```
Comment (acogoluegnes) [inline site/monitoring.xml]: Maybe it's worth mentioning which HTTP endpoint(s) those metrics belong to.

File: site/monitoring.xml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            <tr><td>Message Delivery Rate</td><td><code>message_stats.deliver_get.rate</code></td></tr>
+            <tr><td>Other Message Stats</td><td><code>message_stats.*</code> (see <a href="https://rawcdn.githack.com/rabbitmq/rabbitmq-management/master/priv/www/doc/stats.html">this document</a>)</td></tr>
+          </tbody>
+        </table>
+      </p>
+    </doc:section>
+    <doc:section name="third-party-monitoring-tools">
+      <doc:heading>Third Party Monitoring Tools</doc:heading>
+      <p>
+        The following is an alphabetized list of third-party tools to collect RabbitMQ metrics. These
```
Comment (acogoluegnes) [inline site/monitoring.xml]: "alphabetised"? (RabbitMQ documentation is deeply rooted in British English  :-) ).

**Comment by lukebakken** [comment]
Thanks @acogoluegnes ! Changes have been made and pushed.

File: site/monitoring.xml (hunk 2 above)
Comment (lukebakken) [reply site/monitoring.xml]: Got it, added the following:

```
via the <a href="https://cdn.rawgit.com/rabbitmq/rabbitmq-management/&version-server-tag;/priv/www/api/index.html"><code>api/queues/<i>vhost</i>/<i>qname</i></code></a>
```

**Comment by michaelklishin** [comment]
I will cherry-pick to `live`.

**Pull request merged**